andrew_bird/week_01/jsHomework4.html

```diff
@@ -0,0 +1,208 @@
+	if ( (aSq + bSq) < cSq || (aSq + cSq) < bSq ||(cSq + bSq) < aSq) {
+		return true;
+	} else {
+		return false;
+	}
+}
+//console.log(isEquilateral(triangle));
+//console.log(isIsosceles(triangle));
+console.log(areaTri(triangle));
+console.log(isObtuse(triangle));
+//console.log();
+
+// -------------------------------------------
+
+/*
+JavaScript Bank
+In this homework, you'll create a basic bank in Javascript. The bank has many accounts and the following capabilities that you need to write.
+
+Bank
+
+There is only one bank. This bank has an array of accounts. The bank needs a method that will return the total sum of money in the accounts. It also needs an addAccount method that will enroll a new Account at the bank and add it to the array of accounts. There is no need to create additional functions of the bank to delete accounts, etc.
+
+The bank has many accounts. Accounts should be objects that all share a set of common functionality.
+
+Accounts
+
+Accounts have a current balance and owner's name. You should be able to deposit or withdraw from an Account to change the balance.
+
+There is no need to write a user interface. Make sure functions return values -- you may also have your functions console.log() values to help you see your code working.
+
+You should write a basic story through a series of JavaScript commands that shows that the methods do indeed work as expected: add some accounts, show the total balance, make some deposits and withdrawals, show the new total balance.
+
+Tips
+
+Don't overthink this. Shorter code is probably the answer.
+
+Bonus
+
+Ensure that the accounts cannot have negative values.
+Write a 'transfer' on the bank that allows you to transfer amounts between two accounts.
+
+*/
+
+
+// Create object bank which contins array accounts
+var bank = {name: 'General Assembly Bank',
+	accounts: [],
+
+// function to create an object account and add it to the array of accounts
+	addAccount : function(first, last, amount) {
+		this.accounts.push(new Account(first, last, amount));
+		},
+// function to show the total balance of all accounts by iterating through each object in the array and summing the .amount value.		
+	showTotal : function() {
+			var tot = 0;
+			for(var i = 0; i < this.accounts.length; i++){
+				tot += this.accounts[i].amount;
+				}
+				return tot;
+		}
+	
+	}
+
+
+	//}
+
+// function to create an object account, which has functions to deposit and withdraw form the balance
+function Account(first, last, amount) {
+	this.firstName = first;
+	this.lastName = last;
+	this.amount = amount;
+	this.deposit = function(depositAmount) {
+		this.amount += depositAmount;
+		return "$" + depositAmount + " deposited. Your balance is $" + this.amount;
+	}
+	this.withdraw = function(withdrawAmount) {
+		if(withdrawAmount <= this.amount) {
+		this.amount -= withdrawAmount;
+		return "$" + withdrawAmount + " withdrawn from account."
+		} else {
+			return "your balance is not high enough for that."
+		}
+	}
+}
+
```

Comment:
Very good. What about splitting this huge function Account, that does lot's of things at once on few functions, and if you need just call on them inside of the Account.